docs/faq/general/which-version-control-systems-does-codacy-support.md

```diff
@@ -39,7 +39,7 @@ Codacy supports repositories from the following Git providers:
       <td><p>Codacy Cloud or<br/><a target="_blank" href="https://www.codacy.com/self-hosted">Codacy Self-hosted</a></p>
     </tr>
     <tr>
-      <td><p><strong>GitLab Self-managed</strong><br/>version 12.6.2-ee or later</p></td>
+      <td><p><strong>GitLab Self-managed</strong><br/>version 14.0.0 or later</p></td>
```

Comment:
I checked the GitLab version on gitlab.dev.codacy.org and I saw it is actually 14.8. Should we use that version instead, or is 14.0.0 fine?

![image](https://user-images.githubusercontent.com/60105800/172359945-5c76320e-25af-44ab-b520-084463e8b616.png)

Reply:
Makes sense @lolgab. I updated the version number accordingly. :+1: